### PR TITLE
Remove "preview" note about Python support.

### PR DESCRIPTION
**Pull request by clstokes**
Removes the "preview" note about Python support since it's now GA.

### CODE REVIEW

File: content/docs/get-started/crossguard/_index.md [hunk 1]
```diff
@@ -16,11 +16,7 @@ Often organizations want to empower developers to manage their infrastructure ye
 
 Using Policy as Code, users can express business or security rules as functions that are executed against resources in their stacks. Then using CrossGuard, organization administrators can apply these rules to particular stacks within their organization. When policies are executed as part of your Pulumi deployments, any violation will gate or block that update from proceeding.
 
-Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to stacks written in any language.
-
-{{% notes %}}
-Python support is currently in preview.
-{{% /notes %}}
+Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here](/docs/guides/crossguard/#languages).
```
Comment (justinvp) [inline content/docs/get-started/crossguard/_index.md:19]:
```suggestion
Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here]({{< relref "/docs/guides/crossguard#languages" >}}).
```

File: content/docs/get-started/crossguard/authoring-a-policy-pack.md [hunk 2]
```diff
@@ -11,11 +11,7 @@ aliases: ["/docs/get-started/policy-as-code/authoring-a-policy-pack/"]
 ---
 <!-- markdownlint-disable emphasis ul -->
 
-Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language.
-
-{{% notes %}}
-Python support is currently in preview.
-{{% /notes %}}
+Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here](/docs/guides/crossguard/#languages).
```
Comment (justinvp) [inline content/docs/get-started/crossguard/authoring-a-policy-pack.md:14]:
```suggestion
Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here]({{< relref "/docs/guides/crossguard#languages" >}}).
```

File: content/docs/guides/crossguard/core-concepts.md [hunk 3]
```diff
@@ -10,11 +10,9 @@ menu:
     parent: crossguard
 ---
 
-{{< chooser language "typescript,python" />}}
+Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here](/docs/guides/crossguard/#languages).
```
Comment (justinvp) [inline content/docs/guides/crossguard/core-concepts.md:13]:
```suggestion
Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here]({{< relref "/docs/guides/crossguard#languages" >}}).
```

**Comment by clstokes** [comment]
> LGTM otherwise.

Thanks for the link fixes. I'm still unable to run docs locally without it crashing OOM so was waiting for the Pulumify preview to confirm they worked. 😁

**Comment by clstokes** [comment]
Found a rendering issue on one of the pages so marking this "draft" to prevent merging until I fix.

**Pull request merged**